sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_lease_client.hpp

```diff
@@ -54,22 +54,22 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
      * duration cannot be changed using renew or change.
      * @param options Optional parameters to execute this function.
      * @param context Context for cancelling long running operations.
-     * @return A AcquireShareLeaseResult describing the lease.
+     * @return A AcquireLeaseResult describing the lease.
```

Comment:
```suggestion
     * @return An AcquireLeaseResult describing the lease.
```

Reply:
I'll commit this change once CI's finished